crashes with visible barrier mod

### Minecraft Version

1.21.4

### Skyblocker Version

skyblocker-v5.1.2+1.21.4.jar

### Reproduction Steps

The crash is when I lunch the game with both skyblocker and visible barrier(https://modrinth.com/mod/visiblebarriers) mod enabled it crashes, This has happened recently cause visible barrier has come attached with 2 other mods with the one that seems to be the problem from testing being polymer https://modrinth.com/mod/polymer. 

### Crash Report file and latest.log

```shell
---- Minecraft Crash Report ----
// There are four lights!

Time: 2025-06-24 13:01:16
Description: Initializing game

java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'fabric-data-attachment-api-v1' at 'net.fabricmc.fabric.impl.attachment.sync.AttachmentSync'!
	at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$(FabricLoaderImpl.java:403)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:401)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:52)
	at net.minecraft.class_310.<init>(class_310.java:476)
Caused by: java.lang.ExceptionInInitializerError
	at net.fabricmc.fabric.mixin.attachment.CustomPayloadS2CPacketAccessor.getMaxPayloadSize(CustomPayloadS2CPacketAccessor.java)
	at net.fabricmc.fabric.impl.attachment.sync.AttachmentChange.<clinit>(AttachmentChange.java:58)
	at net.fabricmc.fabric.impl.attachment.sync.s2c.AttachmentSyncPayloadS2C.<clinit>(AttachmentSyncPayloadS2C.java:30)
	at net.fabricmc.fabric.impl.attachment.sync.AttachmentSync.onInitialize(AttachmentSync.java:103)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:399)
	... 2 more
Caused by: java.lang.ClassCastException: class net.azureaaron.hmapi.network.WrappedPacketCodec cannot be cast to class eu.pb4.polymer.networking.impl.ExtCustomPayloadCodec (net.azureaaron.hmapi.network.WrappedPacketCodec and eu.pb4.polymer.networking.impl.ExtCustomPayloadCodec are in unnamed module of loader 'knot' @27c20538)
	at net.minecraft.class_2817.$cbj000$polymer-networking$modifyCodec(class_2817.java:1517)
	at net.minecraft.class_2817.<clinit>(class_2817.java:18)
	... 7 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$2(FabricLoaderImpl.java:403)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:401)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:52)
	at knot//net.minecraft.class_310.<init>(class_310.java:476)

-- Initialization --
Details:
	Modules: 
		ADVAPI32.dll:Advanced Windows 32 Base API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		COMCTL32.dll:User Experience Controls Library:6.10 (WinBuild.160101.0800):Microsoft Corporation
		CRYPT32.dll:Crypto API32:10.0.19041.4522 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTBASE.dll:Base cryptographic API DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTSP.dll:Cryptographic Service Provider API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		DBGHELP.DLL:Windows Image Helper:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		DNSAPI.dll:DNS Client API DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		GDI32.dll:GDI Client DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		IMM32.DLL:Multi-User Windows IMM32 API Client DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		IPHLPAPI.DLL:IP Helper API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		KERNEL32.DLL:Windows NT BASE API Client DLL:10.0.19041.329 (WinBuild.160101.0800):Microsoft Corporation
		KERNELBASE.dll:Windows NT BASE API Client DLL:10.0.19041.388 (WinBuild.160101.0800):Microsoft Corporation
		MSASN1.dll:ASN.1 Runtime APIs:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		MpOav.dll:IOfficeAntiVirus Module:4.18.25050.5 (bcf51ab773be21957c5713cae9cb3adf2fd75bf5):Microsoft Corporation
		NLAapi.dll:Network Location Awareness 2:10.0.19041.388 (WinBuild.160101.0800):Microsoft Corporation
		NSI.dll:NSI User-mode interface DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		NTASN1.dll:Microsoft ASN.1 API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		Ole32.dll:Microsoft OLE for Windows:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		OleAut32.dll:OLEAUT32.DLL:10.0.19041.388 (WinBuild.160101.0800):Microsoft Corporation
		POWRPROF.dll:Power Profile Helper DLL:10.0.19041.4522 (WinBuild.160101.0800):Microsoft Corporation
		PSAPI.DLL:Process Status Helper:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		RPCRT4.dll:Remote Procedure Call Runtime:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		SHCORE.dll:SHCORE:10.0.19041.4522 (WinBuild.160101.0800):Microsoft Corporation
		SHELL32.dll:Windows Shell Common Dll:10.0.19041.329 (WinBuild.160101.0800):Microsoft Corporation
		UMPDC.dll
		USER32.dll:Multi-User Windows USER API Client DLL:10.0.19041.4522 (WinBuild.160101.0800):Microsoft Corporation
		USERENV.dll:Userenv:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		VCRUNTIME140.dll:Microsoft® C Runtime Library:14.36.32532.0:Microsoft Corporation
		VERSION.dll:Version Checking and File Installation Libraries:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		WINHTTP.dll:Windows HTTP Services:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		WINMM.dll:MCI API DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		WS2_32.dll:Windows Socket 2.0 32-Bit DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		Wldp.dll:Windows Lockdown Policy:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		amsi.dll:Anti-Malware Scan Interface:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		apphelp.dll:Application Compatibility Client Library:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		awt.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		bcrypt.dll:Windows Cryptographic Primitives Library:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		bcryptPrimitives.dll:Windows Cryptographic Primitives Library:10.0.19041.264 (WinBuild.160101.0800):Microsoft Corporation
		clbcatq.dll:COM+ Configuration Catalog:2001.12.10941.16384 (WinBuild.160101.0800):Microsoft Corporation
		combase.dll:Microsoft COM for Windows:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		dbgcore.DLL:Windows Core Debugging Helpers:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		extnet.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		fwpuclnt.dll:FWP/IPsec User-Mode API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		gdi32full.dll:GDI Client DLL:10.0.19041.388 (WinBuild.160101.0800):Microsoft Corporation
		gpapi.dll:Group Policy Client API:10.0.19041.4522 (WinBuild.160101.0800):Microsoft Corporation
		java.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		javaw.exe:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		jemalloc.dll
		jimage.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		jli.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		jna8540355099761059019.dll:JNA native library:7.0.2:Java(TM) Native Access (JNA)
		jsvml.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		jvm.dll:OpenJDK 64-Bit server VM:21.0.3.0:Eclipse Adoptium
		kernel.appcore.dll:AppModel API Host:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		lwjgl.dll
		management.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		management_ext.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		msvcp140.dll:Microsoft® C Runtime Library:14.36.32532.0:Microsoft Corporation
		msvcp_win.dll:Microsoft® C Runtime Library:10.0.19041.423 (WinBuild.160101.0800):Microsoft Corporation
		msvcrt.dll:Windows NT CRT DLL:7.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		mswsock.dll:Microsoft Windows Sockets 2.0 Service Provider:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		napinsp.dll:E-mail Naming Shim Provider:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		ncrypt.dll:Windows NCrypt Router:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		net.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		nio.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		ntdll.dll:NT Layer DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		pdh.dll:Windows Performance Data Helper DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		perfos.dll:Windows System Performance Objects DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		perfproc.dll:Windows System Process Performance Objects DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		pnrpnsp.dll:PNRP Name Space Provider:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		profapi.dll:User Profile Basic API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		rasadhlp.dll:Remote Access AutoDial Helper:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		rsaenh.dll:Microsoft Enhanced Cryptographic Provider:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		sechost.dll:Host for SCM/SDDL/LSA Lookup APIs:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		shlwapi.dll:Shell Light-weight Utility Library:10.0.19041.4522 (WinBuild.160101.0800):Microsoft Corporation
		sunmscapi.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		ucrtbase.dll:Microsoft® C Runtime Library:10.0.19041.423 (WinBuild.160101.0800):Microsoft Corporation
		vcruntime140_1.dll:Microsoft® C Runtime Library:14.36.32532.0:Microsoft Corporation
		verify.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		win32u.dll:Win32u:10.0.19041.450 (WinBuild.160101.0800):Microsoft Corporation
		windows.storage.dll:Microsoft WinRT Storage API:10.0.19041.4522 (WinBuild.160101.0800):Microsoft Corporation
		winrnr.dll:LDAP RnR Provider DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		wshbth.dll:Windows Sockets Helper DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		zip.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
Stacktrace:
	at knot//net.minecraft.client.main.Main.main$mixinextras$wrapped$12(Main.java:250)
	at knot//net.minecraft.client.main.Main.mixinextras$bridge$main$mixinextras$wrapped$12$13(Main.java)
	at knot//net.minecraft.client.main.Main.wrapMethod$zoa000$feather$lGEqU(Main.java:517)
	at knot//net.minecraft.client.main.Main.main(Main.java)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at net.digitalingot.rustextension.ProxiedStart.main(ProxiedStart.java:27)

-- System Details --
Details:
	Minecraft Version: 1.21.4
	Minecraft Version ID: 1.21.4
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 21.0.3, Eclipse Adoptium
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Eclipse Adoptium
	Memory: 348781696 bytes (332 MiB) / 671088640 bytes (640 MiB) up to 4227858432 bytes (4032 MiB)
	CPUs: 16
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 7 7700 8-Core Processor              
	Identifier: AuthenticAMD Family 25 Model 97 Stepping 2
	Microarchitecture: Zen 3
	Frequency (GHz): 3.79
	Number of physical packages: 1
	Number of physical CPUs: 8
	Number of logical CPUs: 16
	Graphics card #0 name: AMD Radeon(TM) Graphics
	Graphics card #0 vendor: Advanced Micro Devices, Inc.
	Graphics card #0 VRAM (MiB): 512.00
	Graphics card #0 deviceId: VideoController1
	Graphics card #0 versionInfo: 31.0.22044.1
	Graphics card #1 name: NVIDIA GeForce RTX 3070
	Graphics card #1 vendor: NVIDIA
	Graphics card #1 VRAM (MiB): 8192.00
	Graphics card #1 deviceId: VideoController2
	Graphics card #1 versionInfo: 27.21.14.6627
	Memory slot #0 capacity (MiB): 16384.00
	Memory slot #0 clockSpeed (GHz): 4.80
	Memory slot #0 type: DDR5
	Memory slot #1 capacity (MiB): 16384.00
	Memory slot #1 clockSpeed (GHz): 4.80
	Memory slot #1 type: DDR5
	Virtual memory max (MiB): 40917.89
	Virtual memory used (MiB): 15789.92
	Swap memory total (MiB): 9216.00
	Swap memory used (MiB): 0.00
	Space in storage for jna.tmpdir (MiB): <path not set>
	Space in storage for org.lwjgl.system.SharedLibraryExtractPath (MiB): <path not set>
	Space in storage for io.netty.native.workdir (MiB): <path not set>
	Space in storage for java.io.tmpdir (MiB): available: 563717.06, total: 953237.56
	Space in storage for workdir (MiB): available: 563717.06, total: 953237.56
	JVM Flags: 8 total; -Xmx4023M -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M -XX:ErrorFile=feather/java_error.log
	Fabric Mods: 
		fabric-api: Fabric API 0.119.2+1.21.4
			fabric-api-base: Fabric API Base 0.4.54+b47eab6b04
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.86+b1caf1e904
			fabric-biome-api-v1: Fabric Biome API (v1) 15.0.6+b1c29d8e04
			fabric-block-api-v1: Fabric Block API (v1) 1.0.31+7feeb73304
			fabric-block-view-api-v2: Fabric BlockView API (v2) 1.0.20+9c49cc8c04
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 2.0.8+7feeb73304
			fabric-client-tags-api-v1: Fabric Client Tags 1.1.29+20ea1e2304
			fabric-command-api-v1: Fabric Command API (v1) 1.2.62+f71b366f04
			fabric-command-api-v2: Fabric Command API (v2) 2.2.41+e496eb1504
			fabric-commands-v0: Fabric Commands (v0) 0.2.79+df3654b304
			fabric-content-registries-v0: Fabric Content Registries (v0) 9.1.19+25d1a67604
			fabric-convention-tags-v1: Fabric Convention Tags 2.1.20+7f945d5b04
			fabric-convention-tags-v2: Fabric Convention Tags (v2) 2.14.1+1edcd29104
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.3.6+7feeb73304
			fabric-data-attachment-api-v1: Fabric Data Attachment API (v1) 1.6.2+e99da0f704
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 22.3.1+0f4e5f5504
			fabric-dimensions-v1: Fabric Dimensions API (v1) 4.0.10+7feeb73304
			fabric-entity-events-v1: Fabric Entity Events (v1) 2.0.15+62245bef04
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 4.0.4+a4eebcf004
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.63+7d48d43904
			fabric-item-api-v1: Fabric Item API (v1) 11.3.0+ee91fa1f04
			fabric-item-group-api-v1: Fabric Item Group API (v1) 4.2.2+fcb9601404
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.57+7d48d43904
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.55+df3654b304
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.5.4+bf2a60eb04
			fabric-loot-api-v2: Fabric Loot API (v2) 3.0.38+3f89f5a504
			fabric-loot-api-v3: Fabric Loot API (v3) 1.0.26+203e6b2304
			fabric-message-api-v1: Fabric Message API (v1) 6.0.25+7feeb73304
			fabric-model-loading-api-v1: Fabric Model Loading API (v1) 4.3.0+ae23723504
			fabric-networking-api-v1: Fabric Networking API (v1) 4.4.0+db5e668204
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 18.0.14+38b0d59804
			fabric-particles-v1: Fabric Particles (v1) 4.0.14+7feeb73304
			fabric-recipe-api-v1: Fabric Recipe API (v1) 8.1.1+640e77ae04
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 6.1.11+4a9c1ece04
			fabric-renderer-api-v1: Fabric Renderer API (v1) 5.0.3+50f0feb204
			fabric-renderer-indigo: Fabric Renderer - Indigo 2.0.3+50f0feb204
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.58+73761d2e04
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.1.19+7feeb73304
			fabric-rendering-v1: Fabric Rendering (v1) 10.2.1+0d31b09f04
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 5.0.13+203e6b2304
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 3.1.1+360374ac04
			fabric-screen-api-v1: Fabric Screen API (v1) 2.0.38+7feeb73304
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.118+7feeb73304
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.32+7feeb73304
			fabric-tag-api-v1: Fabric Tag API (v1) 1.0.7+7d48d43904
			fabric-transfer-api-v1: Fabric Transfer API (v1) 5.4.9+efa825c904
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 6.3.2+56e78b9b04
		fabricloader: Fabric Loader 0.16.14
			mixinextras: MixinExtras 0.4.1
		feather: Feather Client release/0a3cebfa
		java: OpenJDK 64-Bit Server VM 21
		minecraft: Minecraft 1.21.4
		modmenu: Mod Menu 13.0.3
			placeholder-api: Placeholder API 2.5.1+1.21.3
		skyblocker: Skyblocker 5.1.2+1.21.4
			com_logisticscraft_occlusionculling: occlusionculling 0.0.8-SNAPSHOT
			hm-api: HM Api 1.0.1+1.21.2
			meteordevelopment_discord-ipc: discord-ipc 1.1
			moe_nea_neurepoparser: neurepoparser 1.6.0
			net_azureaaron_legacy-item-dfu: legacy-item-dfu 1.0.2+1.21.3
			net_azureaaron_networth-calculator: networth-calculator 1.0.5
			org_apache_commons_commons-math3: commons-math3 3.6.1
			org_apache_commons_commons-text: commons-text 1.12.0
			org_eclipse_jgit_org_eclipse_jgit: org.eclipse.jgit 7.0.0.202409031743-r
			yet_another_config_lib_v3: YetAnotherConfigLib 3.6.2+1.21.4-fabric
				com_twelvemonkeys_common_common-image: common-image 3.12.0
				com_twelvemonkeys_common_common-io: common-io 3.12.0
				com_twelvemonkeys_common_common-lang: common-lang 3.12.0
				com_twelvemonkeys_imageio_imageio-core: imageio-core 3.12.0
				com_twelvemonkeys_imageio_imageio-metadata: imageio-metadata 3.12.0
				com_twelvemonkeys_imageio_imageio-webp: imageio-webp 3.12.0
				org_quiltmc_parsers_gson: gson 0.2.1
				org_quiltmc_parsers_json: json 0.2.1
		visiblebarriers: Visible Barriers 2.0.20
			cardinal-components-base: Cardinal Components API (base) 6.2.2
			cardinal-components-entity: Cardinal Components API (entities) 6.2.2
			polymer-core: Polymer (Core) 0.11.6+1.21.4
				polymer-networking: Polymer (Networking) 0.11.6+1.21.4
					polymer-common: Polymer (Common) 0.11.6+1.21.4
						packet_tweaker: Packet Tweaker 0.6.0-pre.1+1.21.2-pre3
				polymer-registry-sync-manipulator: Polymer (Registry Sync Manipulator) 0.11.6+1.21.4
	Launched Version: 1.21.4-feather
	Backend library: LWJGL version 3.3.3-snapshot
	Backend API: Unknown
	Window size: <not initialized>
	GFLW Platform: <error>
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: <disabled>
	Is Modded: Definitely; Client brand changed to 'Feather Fabric'
	Universe: 404
	Type: Client (map_client.txt)
	Locale: en_AU
	System encoding: Cp1252
	File encoding: UTF-8
	CPU: <unknown>
```

### Additional context

_No response_

--- a/gradle.properties
+++ b/gradle.properties
@@ -12,11 +12,11 @@ fabric_api_version=0.140.0+1.21.11
 
 # Minecraft Mods
 ## Dandelion
-dandelion_version=1.0.0-alpha.14+1.21.11
+dandelion_version=1.0.0-alpha.15+1.21.11
 ## YACL (https://github.com/isXander/YetAnotherConfigLib)
 yacl_version=3.8.1+1.21.11
 ## HM API (https://github.com/AzureAaron/hm-api)
-hm_api_version=1.0.1+1.21.2
+hm_api_version=1.0.2+1.21.11
 ## Mod Menu (https://modrinth.com/mod/modmenu/versions)
 mod_menu_version = 17.0.0-alpha.1
 ## REI (https://modrinth.com/mod/rei/versions?l=fabric)

